lib/rex/logging/log_dispatcher.rb

```diff
@@ -176,5 +198,23 @@ def get_log_level(src)
   $dispatcher.get_level(src)
 end
 
+def get_global_log_level(src)
+  level = get_log_level(src)
+
+  # If the source has no associated log_level, the default log level is used
+  unless level
+    level = LEV_3
+  end
+
+  level
+end
```

Comment:
We could move this log into the log dispatcher:

```ruby
  def get_level(src)
    log_levels.fetch(src, LEV_0)
  end
```

And we could delete this:

```suggestion
```

Reply:
Implemented